handler/src/main/java/io/netty/handler/ssl/SslContextBuilder.java

```diff
@@ -48,6 +51,17 @@ public static SslContextBuilder forServer(File keyCertChainFile, File keyFile) {
     /**
      * Creates a builder for new server-side {@link SslContext}.
      *
+     * @param key a PKCS#8 private key
+     * @param keyCertChain the X.509 certificate chain
+     * @see #keyManager(PrivateKey, X509Certificate[])
+     */
+    public static SslContextBuilder forServer(PrivateKey key, X509Certificate... keyCertChain) {
+        return new SslContextBuilder(true).keyManager(key, keyCertChain);
```

Comment:
Checking in our code (or as early as possible) is almost always good in my opinion because 1) the stack trace is easier to read and 2) we can provide more useful exception message (e.g. correct parameter name)